src/sqlfluff/rules/capitalisation/CP01.py

```diff
@@ -145,7 +145,7 @@ def _handle_segment(self, segment: BaseSegment, context: RuleContext) -> LintRes
             first_letter_is_lowercase = False
 
         if first_letter_is_lowercase:
-            refuted_cases.update(["upper", "capitalise", "pascal"])
+            refuted_cases.update(["upper", "capitalise", "pascal", "snake"])
```

Comment:
Can you add a comment here about how snake case cannot be inferred, because it presents the same as lower

Reply:
added! thank you for the input!